xls/modules/rle/rle_enc.x

```diff
@@ -66,8 +66,8 @@ struct RunLengthEncoderState<SYMBOL_WIDTH: u32, COUNT_WIDTH: u32> {
 
 // RLE encoder implementation
```

Comment:
curious why we need the original implementation to change?

Reply:
This PR modifies existing interfaces for symbols and encoded pairs.
It adds parameter for number of symbols/pairs in single transaction.
It also adds symbol_valids.
This is why I had to update existing implementations.